hw8/task1.py

```diff
@@ -0,0 +1,51 @@
+from collections import defaultdict
+
+
+class KeyValueStorage:
+    """
+        This is a wrapper class  for some key-value storage file.
+         File consists rows,  each other is represented as key and value separated by = symbol.
+
+         This class has its(storage) keys and values accessible as collection items and as attributes.
+    """
+
+    def __init__(self, file_path):
+        """
+        :param file_path: type: str, takes path to file which is a key-value storage file.
+        """
+        self.file_path = file_path
+        self.item = defaultdict(int)
+        with open(self.file_path, 'r') as fi:
+            for line in fi:
+                k, v = line.strip('\n').split('=')
+                if k.isnumeric() and k != v:
```

Comment:
what if k has blank space?

Reply:
k != v what does the statement checks?

Reply:
why can't I have `1=1`

Reply:
Fix: 
add replace for blank space, 
'k != v' it checked if  file has lines like 1 = 2 and if it has -> ValueError
and so you can have 1=1, because of this statement 'k != v '